src/ui/page/booking/choose-date.html

```diff
@@ -38,7 +38,7 @@ <h1 class="heading-large">
                                 <td>{{course.location}}</td>
                                 <td>&pound;{{course.price}}</td>
                                 <td>
-                                    {{#if availability.id == selectedDate}}
+                                    {{#if availability.id == (selectedDate + 1)}}
```

Comment:
`selectedDate` shouldn't be treated as a sequential index rather it's an ID - if there are dates that are too old to be shown then the first id won't be '0'. I intentionally defaulted the selectedDate to -1 in the controller so we don't select anything. As selecting the first means when a user clicks "Change date" their selection is reset to the first, better to not select anything than to select the wrong thing. 

If we want to select a date we need to sort going back to the page, otherwise we should leave it unselected.